pkgs/by-name/li/libtommath/package.nix

```diff
@@ -14,21 +14,24 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-KWJy2TQ1mRMI63NgdgDANLVYgHoH6CnnURQuZcz6nQg=";
   };
 
-  nativeBuildInputs = [ libtool ];
-
   postPatch = ''
-    substituteInPlace makefile.shared --replace glibtool libtool
-    substituteInPlace makefile_include.mk --replace "shell arch" "shell uname -m"
+    substituteInPlace makefile.shared \
+      --replace-fail glibtool libtool \
+      --replace-fail libtool "${lib.getExe libtool}"
+    substituteInPlace makefile_include.mk \
+      --replace-fail "gcc" "${stdenv.cc.targetPrefix}cc"
   '';
 
   preBuild = ''
     makeFlagsArray=(PREFIX=$out \
+      CC=${stdenv.cc.targetPrefix}cc \
       INSTALL_GROUP=$(id -g) \
       INSTALL_USER=$(id -u))
   '';
 
   makefile = "makefile.shared";
 
+  env.CC = "${stdenv.cc.targetPrefix}cc";
```

Comment:
```suggestion
```

builds for me w/o setting `env.CC`, since it's already in makeFlagsArray above.

Reply:
Done. I had to override libtool to fix `libtommath.override { stdenv = clangStdenv; }`. Let me know if it makes sense.

Reply:
oh, so maybe keep the `env.CC` and drop the `makeFlags` CC entry instead?

if it really needs both CC assignments that's better than an stdenv override and let's just go with your original patch; it's just a little surprising. which host are you building for & from what type of build machine?

Reply:
I am building on x64 Linux for aarch64 Linux. I think setting env.CC had no effect on libtool, but I will check again.
For my original patch I didn't check clangStdenv.